source/geod24/LocalRest.d

```diff
@@ -701,25 +697,32 @@ public final class RemoteAPI (API) : API
                         },
                         (Response res) {
                             if (!isSleeping())
+                            {
                                 handle(res);
+                            }
                             else if (!control.drop)
-                                await_msgs ~= Variant(res);
+                            {
+                                scheduler.spawn({
+                                    while (isSleeping())
```

Comment:
Does this work ? If so, this could be its own commit / pull requests, right ? Or does it depends on a specific behavior of the new concurrency module ?

I see two potential issues here:
- A node could be made to re-sleep, which would turn `isSleeping` back to `true`, before all fibers that `yield` get processed.
- Spawning a new fiber instead of just keeping a variant is much more memory usage / CPU.